scripts/add_ts_data_bidmatrixinfo.py

```diff
@@ -0,0 +1,84 @@
+###this script adds time series data to the bid matrix information objects in the powerops client
+
+from cognite.client.data_classes import TimeSeries
+from cognite.client.exceptions import CogniteNotFoundError
+from cognite.powerops import PowerOpsClient
+import random
+import datetime
+
+power = PowerOpsClient.from_settings()
+cdf = power.cdf
+
+exid_hourly = "emilies_points"  # exid_hourly = "wim_ts_hourly"
+exid_empty = "wim_ts_empty"
+tsList = [exid_hourly, exid_empty]
```

Comment:
**note:** it's most common to use `snake_case` for naming variables in python, and it's also best practice to pick a pattern and stick to it for consistency, so don't mix `snake_case` and `camelCase` for variable/function names (things like classes are usually `CamelCase`, these standards are all documented in pythons [PEP 8 style guide](https://peps.python.org/pep-0008/#naming-conventions)